x/stakeibc/types/message_rebalance_validators.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 const TypeMsgRebalanceValidators = "rebalance_validators"
+const MIN_NUM_REBALANCE = 1
+const MAX_NUM_REBALANCE = 10
```

Comment:
If you set these constants here, shouldn't you reference them on line 55, rather than leaving the hardcoded values?